test/org/sagebionetworks/bridge/models/schedules/ScheduleTest.java

```diff
@@ -0,0 +1,58 @@
+package org.sagebionetworks.bridge.models.schedules;
+
+import static org.junit.Assert.*;
+
+import nl.jqno.equalsverifier.EqualsVerifier;
+import nl.jqno.equalsverifier.Warning;
+
+import org.joda.time.DateTime;
+import org.joda.time.LocalTime;
+import org.joda.time.Period;
+import org.junit.Test;
+import org.sagebionetworks.bridge.json.BridgeObjectMapper;
+
+import com.google.common.collect.Lists;
+
+public class ScheduleTest {
+
+    @Test
+    public void equalsHashCode() {
+        EqualsVerifier.forClass(Schedule.class).suppress(Warning.NONFINAL_FIELDS).allFieldsShouldBeUsed().verify();
+    }
+    
+    @Test
+    public void canRountripSerialize() throws Exception {
```

Comment:
Given that the JSON is part of our external interface, I feel we should define the JSON structure and test against it.